python/ray/experimental/workflow/examples/comparisons/argo/dag_workflow.py

```diff
@@ -0,0 +1,17 @@
+from ray import workflow
+
+
+@workflow.step
+def echo(msg: str, *deps) -> None:
+    print(msg)
+
+
+if __name__ == "__main__":
+    workflow.init()
+
+    A = echo.options(name="A").step("A")
```

Comment:
right now I'm using `step_name`, do you think it should be updated to `name`?

Reply:
I think step name is preferable to be consistent with other APIs (also, it's redundant since it's clear what is being named).